spec/integration/util/windows/security_spec.rb

```diff
@@ -553,6 +604,32 @@ def grant_everyone_full_access(path)
           # end
         end
 
+        describe "#mode=" do
+          # setting owner to SYSTEM requires root
+          it "should round-trip all 54 modes that do not require deny ACEs, when simulating a SYSTEM scheduled task" do
+            # The owner and group for files/dirs created, when running as a Scheduled Task as Local System are
+            # Owner = SYSTEM
+            # Group = SYSTEM
+            winsec.set_group(sids[:system], path)
+            winsec.set_owner(sids[:system], path)
```

Comment:
It turned out that setting an owner of `SYSTEM` does require special perms, requiring this test to be guarded.

[SetSecurityInfo](https://docs.microsoft.com/en-us/windows/desktop/api/aclapi/nf-aclapi-setsecurityinfo) will fail with `INVALID_OWNER` / `This security ID may not be assigned as the owner of this object.`